lib/gateway/pipelines.js

```diff
@@ -186,7 +154,7 @@ function configurePipeline (pipelinePoliciesConfig, config) {
   return router;
 }
 
-function validateGatewayConfig (config) {
+function normalizeGatewayConfig (config) {
```

Comment:
can schema require gatewayConfig.pipelines or  gatewayConfig.pipeline ?
 
I just don't understand the reason to leave true\false login